src/Compilers/Core/Portable/CommandLine/AnalyzerConfig.cs

```diff
@@ -222,6 +222,12 @@ internal sealed class Section
             /// </summary>
             public static StringComparison NameComparer { get; } = StringComparison.Ordinal;
 
+            /// <summary>
+            /// Used to compare <see cref="Name"/>s of sections. Specified by editorconfig to
+            /// be a case-sensitive comparison.
+            /// </summary>
+            internal static IEqualityComparer<string> NameEqualityComparer { get; } = StringComparer.Ordinal;
```

Comment:
```suggestion
            public static IEqualityComparer<string> NameEqualityComparer { get; } = StringComparer.Ordinal;
```

Generally I dislike using `public` inside `internal` types but that is the style of this part of the code base.